src/app/settings/keyboard-mapping-settings/components/edit-keyboard-mapping/edit-keyboard-mapping.component.html

```diff
@@ -55,11 +52,16 @@
 </sofie-button-group>
 
 <ng-template #actionArguments>
-  <div class="selected-action mb-1"><ng-container i18n>action-triggers.selected-action.label</ng-container> <sofie-form-required-mark></sofie-form-required-mark></div>
   <sofie-row>
     <sofie-action-box [action]="selectedAction!"></sofie-action-box>
     <div class="selected-action-description ps-1">
       {{ selectedAction!.description }}
     </div>
   </sofie-row>
 </ng-template>
+
+<ng-template #noSelectedAction>
+  <sofie-row>
+    <p i18n>action-triggers.no-selected-action.label</p>
```

Comment:
What's the reasing for using a `p`? 
The `p` puts quite a bit space between the title and the message
![image](https://github.com/tv2/ng-sofie/assets/23049647/f1c61782-2295-43cb-8742-ba00dddb7810)
Using a `span` gives it a nicer "distance"
![image](https://github.com/tv2/ng-sofie/assets/23049647/87d3be75-742c-4898-bd56-fb62a961a6b0)

Reply:
@LindvedKrvang I use the `p`, because I like the spacing, but now is changed to `span`